apps/docs/content/guides/auth/auth-anonymous.mdx

```diff
@@ -0,0 +1,130 @@
+---
+id: 'auth-anonymous'
+title: 'Anonymous Sign-Ins'
+subtitle: 'Create and use anonymous users to authenticate with Supabase'
+---
+
+Enabling anonymous sign-ins allows you to reduce the onboarding friction for your application. New users are able to try out your application before actually signing up for it. For example, you can use anonymous sign-ins to to build e-commerce websites to allow users to add items to their shopping cart without having to sign in and only prompt them to do so when they are about to checkout.
+
+<Admonition type="note" label="Difference between an anonymous user and the anon key">
+
+An anonymous user is defined as a user that is created through the `signInAnonymously()` method. The anonymous user will have an `is_anonymous: true` claim in the access token JWT. Like permanent users, an anonymous user will use the `authenticated` role in your database. This is different from the `anon` key in your project, which has no user ID and uses the `anonymous` role in your database.
+
+</Admonition>
+
+To get started, [enable anonymous sign-ins](/dashboard/project/_/auth/providers) for your Supabase Project.
+
+<Admonition type="note" label="Self hosting">
```

Comment:
Maybe move this to CLI docs?

Reply:
I vote for leaving it in here, it's also useful for local dev and a lot of people aren't aware that this is how to change configs for local. Maybe change the label to `Self hosting and local development`?